src/pages/my_registrations/MyRegistrations.tsx

```diff
@@ -44,6 +53,36 @@ export const MyRegistrations = ({ selectedUnpublished, selectedPublished }: MyRe
       return 0;
     });
 
+  const deleteDraftRegistrationMutation = useMutation({
+    mutationFn: async () => {
+      const deletePromises = draftRegistrations.map(async (registration) => {
+        const identifierToDelete = getIdentifierFromId(registration.id);
+        await deleteRegistration(identifierToDelete);
+      });
+
+      await Promise.all(deletePromises);
+    },
+    onSuccess: () => {
+      dispatch(
+        setNotification({
+          message: t('feedback.success.delete_registration'),
+          variant: 'success',
+        })
+      );
+    },
+    onError: () =>
+      dispatch(
+        setNotification({
+          message: t('feedback.error.delete_registration'),
+          variant: 'error',
+        })
+      ),
+    onSettled: () => {
+      registrationsQuery.refetch();
+      setShowDeleteModal(false);
+    },
+  });
```

Comment:
Usikker her, men gir det mening at dialogen fortsatt er åpen så lenge kallet feiler? Slik at man kan prøve på nytt med en gang. Mulig det blir litt rart om det bare feiler for 2 av 50, men de tilfellene blir rare uansett tror jeg

```suggestion
    onSuccess: () => {
      dispatch(
        setNotification({
          message: t('feedback.success.delete_registration'),
          variant: 'success',
        })
      );
      setShowDeleteModal(false);
    },
    onError: () =>
      dispatch(
        setNotification({
          message: t('feedback.error.delete_registration'),
          variant: 'error',
        })
      ),
    onSettled: () => registrationsQuery.refetch(),
  });
```

Reply:
litt usikker på hvilken handler som brukes om noen kall går fint mens andre feiler.. har du testet?

Reply:
kom på nå at om 50 deletes trigger 50 onSettled er vi like langt som i sted, men jeg tipper den trigges bare en gang, og at enten onSuccess eller onError trigges bare en gang

Reply:
HAr ikke fått testet når noe går galt, men det virker som om de kun trigges en gang.